### DISCUSSION TITLE
How does Gatsby release process work?

### DISCUSSION THREAD

**Discussion by hhimanshu** [gatsbyjs/gatsby#30055] [Help]
I am curious to learn how Gatsby releases new software versions? What does your process look like? I tried looking up on the gatsby documentation and did not find anything there. Could you folks please help understand?

**Comment by LekoArts** [comment]
Do you have any specific questions? This would make it easier to answer :)

**Reply by hhimanshu** [reply]
Yes, I am interested in knowing how branching, CI/CD works, how/when release candidates are cut? Also, are there situations for hotfixes and how are they handled?

Does it make sense @LekoArts?

**Comment by LekoArts** [comment]
So this is the current state, this might be outdated information in half a year already.

- We do a minor every two weeks
  - We do a branch cut 5 days before the release date
  - We maintain the release branch two weeks (and backport fixes/do hotfixes)
- A nightly/next version is always the current `master` and when we publish it
- So far no CI involved in releasing stuff, it's all manual
- Backports/Patches are done ad-hoch
